### PR TITLE
SSDAlignment and MomReco Modules

### PR DESCRIPTION
**Pull request by JMirabito**
Update to SSDAlignment module, works for x/y and u/w sensors. Changed definition of u/w sensors to match a positive pi/2 rotation angle of the x/y plane. 

Added Momentum reconstruction module which uses the first order alignment constant to estimate momentum of incoming particles.

Added alignment constants from first order alignment module as "SSDAlign_1c_firstorder.txt" in ConstBase/Align.

### CODE REVIEW

File: SSDAlignment/SSDAlign.h [hunk 1]
```diff
@@ -9,11 +9,13 @@
 #include <vector>
 #include <stdint.h>
 #include <iostream>
+#include "TVector3.h"
```
Comment (gavinsdavies) [inline SSDAlignment/SSDAlign.h:12]: is this needed? i don't see where it would be called

File: SSDAlignment/ssdalignment_job.fcl [hunk 2]
```diff
@@ -2,13 +2,22 @@
...
+#include "DetGeoMap.fcl"
+#include "Align.fcl"
 #include "FillDataQuality.fcl"
 
 process_name: SSDAlignment
 
 services:
 {
   TFileService: { fileName: "ssd_align_hist_r%r_s%04s.root" closeFileFast: false }
+  ChannelMapService: @local::standard_channelmap
```
Comment (gavinsdavies) [inline SSDAlignment/ssdalignment_job.fcl:16]: these are all included by the @table::standard_services from [Services.fcl](https://github.com/EMPHATICSoft/emphaticsoft/blob/main/Utilities/Services.fcl) so shouldn't be needed here

**Review by gavinsdavies — APPROVED** [review]
in the interests of progress, approve.

File: SSDAlignment/SSDAlign.h (hunk 1 above)
Comment (JMirabito) [reply SSDAlignment/SSDAlign.h]: They are unnecessary, removed.

File: SSDAlignment/ssdalignment_job.fcl (hunk 2 above)
Comment (JMirabito) [reply SSDAlignment/ssdalignment_job.fcl]: Also removed these excess lines.

**Pull request merged**